src/domUpdates.js

```diff
@@ -153,17 +164,26 @@ function showSearchResults() {
     } else {
       searchedRecipes.forEach(recipe => {
         allRecipeDisplay.innerHTML += `
-          <div class="recipe-wrapper">
-            <img id="${recipe.name}" src="${recipe.image}" class="recipe" alt="${recipe.name}">
-            <div class="recipe-info">
-              <p>${recipe.name}</p>
+          <div class="recipe-wrapper recipe" tabindex="0" id="${recipe.name}" >
+            <img src="${recipe.image}" class="recipe" alt="${recipe.name}">
+            <div class="recipe-info recipe">
+              <p class="recipe">${recipe.name}</p>
             </div>
           </div>`;
       });
     };
   });
 };
 
+const checkCurrentSavedRecipes= event => {
```

Comment:
Very nice function! I've never used the some method before, but definitely see how I could use it in the future now!